sdk/containerregistry/azure-containerregistry/azure/containerregistry/_helpers.py

```diff
@@ -30,6 +30,11 @@
 ]
 OCI_MANIFEST_MEDIA_TYPE = "application/vnd.oci.image.manifest.v1+json"
 
+# Supported audiences
+AZURE_RESOURCE_MANAGER_CHINA = "https://management.chinacloudapi.cn"
+AZURE_RESOURCE_MANAGER_GOVERNMENT = "https://management.usgovcloudapi.net"
```

Comment:
We do we have these other two constants? We don't need them right?

Reply:
They are used in tests to verify when audience doesn't match the cloud that the resources are in will fail.

Reply:
Has removed the constants, change to retrieve them directly from authorities.

Reply:
Keep these other two constants - 1. convenient for customers to use, 2. used in testcase.py

Reply:
None of these are currently exposed publicly, so customers can't use them anyway.
I'm also not sure we want to expose them publicly - are the other language SDKs doing so? And if so, what are they called?

Reply:
The audience that customer provided should match the cloud, so these values will be used when customer is using China cloud or Us Gov cloud. We are using the same name as in Java: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/models/ContainerRegistryAudience.java#L14 cc: @lmolkova

Reply:
Have those names been through archboard review? I believe we have terminology for these in other SDKs and I'd prefer we have consistent naming. Alternatively, we can just not make these public for now - the strings themselves are already documented in the docstring.

Reply:
Got response from ACR v-team that we are figuring out the audience story with ACR folks. I removed these public constants for now.